### PR TITLE
Question Five of the quiz

### PR DESCRIPTION
**Pull request by utkarshducoder**
please review my Question five page

live link: https://quizzyfuy.netlify.app/quiz%20html/quiz5.html

Following are the features of this page:

- nav: this feature is made to add headings.
- section: this contains the main documentation part of the question.
- anchor tag: this feature is added to add the link for the next question.

### CODE REVIEW

File: question5/quiz5.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.container{
+    display: flex;
+    background-color: black;
+}
+.container div{
+    margin: 20px;
+    width: 100%;
+    color: #fff;
```
Comment (khushijohri2001) [inline question5/quiz5.css:30]: use var here

File: question5/quiz5.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    padding-left: 20%;
+}
+
+#option1{
+    font-size:2rem;
+    border-radius:12px;
+    cursor: pointer;
+    text-align: center;
+    padding:20px;
+    color:#fff;
```
Comment (khushijohri2001) [inline question5/quiz5.css:41]: use var here

File: question5/quiz5.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    width: 35rem;
+    height: 4rem;
+}
+#option2{
+    font-size:2rem;
+    border-radius:12px;
+    cursor: pointer;
+    text-align: center;
+    padding:20px;
+    color:#fff;
```
Comment (khushijohri2001) [inline question5/quiz5.css:53]: use var here

**Review by khushijohri2001 — APPROVED** [review]

File: question5/quiz5.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (i-am-a-codegirl) [inline question5/quiz5.html]: give proper title

**Review by i-am-a-codegirl — APPROVED** [review]
Good to merge

**Pull request merged**